docs/dev_guide/frequently-asked-questions-faq.rst

```diff
@@ -11,6 +11,7 @@ Frequently Asked Questions
 * `How do I use the migrated module file in the new project?`_
 * `Is the memory space allocated by sycl::malloc_device, sycl::malloc_host, and dpct::dpct_malloc initialized?`_
 * `How do I migrate CUDA\* source code that contains CUB library implementation source code?`_
+* `How do I fix the hang issue of sycl code caused by work group level synchronization like group barrier used in conditional statement?`_

```

Comment:
How do I fix the issue of SYCL* code hanging due to work group level synchronization, such as a group barrier used in a conditional statement?

Reply:
Add a backslash before*, It seem that "*" would be used as a control character.
Please double conform.